aries-site/src/data/structures/components.js

```diff
@@ -46,6 +47,24 @@ export const components = [
     relatedContent: ['Button', 'Typography', 'Forms'],
     tags: ['hyperlinks', 'links', 'linking', 'navigation'],
   },
+  {
+    name: 'Avatar',
+    category: 'Visualizations',
+    description: 'An Avatar draws attention to users so that they can efficiently identify a person, company, or entity.',
+    seoDescription: 'An Avatar draws attention to users so that they can efficiently identify a person, company, or entity.',
+    sections: [
+      'Anatomy',
+      'Accessibility',
+      'When to use Avatars',
+      'Variants'
```

Comment:
Add the subsections also of DataTable, Card, PageHeader

Reply:
done